### PR TITLE
Feature2

### PR DESCRIPTION
**Pull request by alwill**

### CODE REVIEW

File: app/api/views.py [hunk 1]
```diff
@@ -13,5 +13,13 @@ def process_coords():
...
+    data = json.loads(request.form.get("coords"))
+    hsegs = []
+
+    for polygon in data:
+    	segment = polygon.get("b")
+    	hsegs.append(segment)
+
+    session["polygons"] = hsegs
+    
+    return "{}".format(session)
```
Comment (jflowaa) [inline app/api/views.py]: - Region isn't being created for each polygon
- You're returning the session

I'm thinking there should be an entry in `session` for the regions that are generated from `create_segments` like how `polygon` entry is. `create_segments` should be refactored to be called `create_region`.

File: app/api/views.py [hunk 2]
```diff
@@ -13,5 +13,13 @@ def process_coords():
...
 
-    data = json.loads(request.form.get("coords")).get("b")
-    return "{}".format(create_segments(data))
+    data = json.loads(request.form.get("coords"))
+    hsegs = []
+
+    for polygon in data:
+    	segment = polygon.get("b")
+    	hsegs.append(segment)
+
```
Comment (jflowaa) [inline app/api/views.py]:
``` python
session["regions"] = []
for hseg in hsegs:
    session.get("regions").append(create_segment(hseg))
```

File: app/api/views.py (hunk 1 above)
Comment (jflowaa) [inline app/api/views.py]:
``` python
    return "{}".format(session.get("regions"))
```

File: app/static/google_maps_drawer.js [hunk 3]
```diff
@@ -27,18 +28,21 @@ function initialize() {
...
+        });
     });
-}
-
-function overlayClickListener(overlay) {
-	google.maps.event.addListener(overlay, "mouseup", function(event){
-    $('#coords').val(overlay.getPath().getArray());
-	});
-}
+}
```
Comment (jflowaa) [inline app/static/google_maps_drawer.js]: File doesn't end on a newline.

File: app/api/views.py (hunk 2 above)
Comment (alwill) [reply app/api/views.py]: Ya i deleted that on accident. When I changed around the session storage

File: app/api/views.py [hunk 4]
```diff
@@ -13,5 +13,15 @@ def process_coords():
...
     """
 
-    data = json.loads(request.form.get("coords")).get("b")
-    return "{}".format(create_segments(data))
+    data = json.loads(request.form.get("coords"))
+    hsegs = []
+    polygons = []
+    for polygon in data:
+    	segment = polygon.get("b")
+        polygons.append(segment);
```
Comment (jflowaa) [inline app/api/views.py]: semi-colon and not 4 space indent

File: app/api/views.py [hunk 5]
```diff
@@ -13,5 +13,15 @@ def process_coords():
...
+    	hsegs.append(create_segments(segment))
+
+    session["polygons"] = polygons
+    session["regions"] = hsegs
+    
+    return "{}".format(session["polygons"])
```
Comment (jflowaa) [inline app/api/views.py]: Returns `polygons`. The more I think about it. This doesn't need to actually return data. Just a response for success.

``` python
    return "", 200
```

File: app/api/views.py (hunk 4 above)
Comment (jflowaa) [reply app/api/views.py]: Line 20 and 22 are using tabs and not spaces

**Pull request merged**